.travis.yml

```diff
@@ -4,16 +4,16 @@ language: python
 python:
 - "2.7"
 - "3.7"
+env:
+- RUNTIME=singularity POPPER_TEST_MODE=with-git
+- RUNTIME=singularity POPPER_TEST_MODE=without-git
+- RUNTIME=docker POPPER_TEST_MODE=with-git
+- RUNTIME=docker POPPER_TEST_MODE=without-git
```

Comment:
since `with-git` and `without-git` test the same functionality regardless of the runtime, we can drop lines 9-10